main.go

```diff
@@ -71,6 +65,8 @@ func main() {
 
 	processor := processor.NewProcessor()
 	go processor.CreateUpdatePolicyReports(fetchPolicyReports, ret, hubID)
+	//start triggering reports for clusters
+	go ret.FetchClusters(monitor, fetchClusterIDs)
```

Comment:
Suppose this works fine as its constantly running, but  feels like a weird order that it would be after the `RetrieveCCXReport` & `CreateUpdatePolicyReports` functions.

Reply:
Though we call in different order , the order  of process execution in runtime is not guaranteed  for our go routines . I only pushed it down , because we want to start fetching only if we have content to lookup . @zlayne